diff --git a/content/posts/2024-10-05-arkansas-traveller-100/index.mdx b/content/posts/2024-10-05-arkansas-traveller-100/index.mdx
index 4dcd631..c95a7e0 100644
--- a/content/posts/2024-10-05-arkansas-traveller-100/index.mdx
+++ b/content/posts/2024-10-05-arkansas-traveller-100/index.mdx
@@ -43,9 +43,9 @@ I wrapped up the last training week by tapering the distance while slightly incr
 
 ### Before the race
 
-Unlike my last few 100 mile races, I thought it would be nice to have a pacer/crew join for the experience. A friend from the UK, Phil, was kindly volunteered, and it was a great stroke of luck as he’s an experienced crew member and pacer, having come 10th-place at the Centurion Thames Path 100.
+Unlike my last few 100 mile races, I thought it would be nice to have a pacer/crew join for the experience. A friend from the UK, Phil, kindly volunteered, and it was a great stroke of luck as he’s an experienced crew member and pacer, having come 9th-place at the Centurion Thames Path 100.
 
-e both made our way to Little Rock on the Wednesday before the race, well at least I did. Phil had some rotten luck and got stuck in Dallas after his UK flight was delayed. My trip from San Francisco went smoothly, and I arrived early. After checking in at our Airbnb I went for a short shakeout run to keep my running streak alive and enjoyed a delicious burger at a local smash burger joint. I then treated myself to a 10-hour sleep before another shakeout run around the Little Rock river and bridges, ending at a fantastic coffee shop to kick off my carbo-loading. Phil arrived shortly after, and we did some exploring together.
+We both made our way to Little Rock on the Wednesday before the race, well at least I did. Phil had some rotten luck and got stuck in Dallas after his UK flight was delayed. My trip from San Francisco went smoothly, and I arrived early. After checking in at our Airbnb I went for a short shakeout run to keep my running streak alive and enjoyed a delicious burger at a local smash burger joint. I then treated myself to a 10-hour sleep before another shakeout run around the Little Rock river and bridges, ending at a fantastic coffee shop to kick off my carbo-loading. Phil arrived shortly after, and we did some exploring together.
 
 I managed around 700 grams of carbs that day, eating:
 
@@ -100,19 +100,19 @@ As I made my way to the Lake Winona aid station the trail was technical but as i
 ###  To Crew Spot 3 (30.9 to 47.6 miles)
 #### Aid Stations: Lake Winona to Powerline
 
-Arriving at the next Aid station Phil was waiting, and we made another quick transition with him grabbing me snacks, refilliong my ice bandanan and my gels. He had also filled my water bottles with ice-cold water, which was glorious. I left quickly and after a brief descent started climbing up to Pigtrail - where I topped up with more ice. I kept of pushing through ascending and descending as swifly as my legs would carry me, hindered mainly by the occasional bashed toe, ouch. 
+Arriving at the next Aid station Phil was waiting, and we made another quick transition with him grabbing me snacks, refilling my ice bandanan and my gels. He had also filled my water bottles with ice-cold water, which was glorious. I left quickly and after a brief descent started climbing up to Pigtrail - where I topped up with more ice. I kept on pushing through ascending and descending as swiftly as my legs would carry me, hindered mainly by the occasional bashed toe, ouch.
 
-Arrinvg at Bahama Mama I had a fun chat with the friendly Aid Station folks, before pushing on. Most of the course had been hard-packed dirt, so it was noticable that this section started with a bit of grassy, I was extra vigilant for snakes after my earlier chat but I none were soptted. It was at this point that I started walking more of the hills. The heat of the day and the relentelsss pushing had got to me. Or perhaps my lower calorie intake in the last section was the culprit. Time for a new mantra, "walking isn't free", I really liked this one and used it throughout the rest of the race. I would chant this and then remind myself, (a) walking is slower (b) it still requires effort (c) more time on feet overall - it really helped me to overcome the brain telling me to walk. 
+Arriving at Bahama Mama I had a fun chat with the friendly Aid Station folks, before pushing on. Most of the course had been hard-packed dirt, so it was noticeable that this section started with a bit of grass, I was extra vigilant for snakes after my earlier chat but I none were spotted. It was at this point that I started walking more of the hills. The heat of the day and the relentless pushing had got to me. Or perhaps my lower calorie intake in the last section was the culprit. Time for a new mantra, "walking isn't free", I really liked this one and used it throughout the rest of the race. I would chant this and then remind myself, (a) walking is slower (b) it still requires effort (c) more time on feet overall - it really helped me to overcome the brain telling me to walk.
 
 During this section a few folks were in the pain cave with the very high heat and I passed them on ascents or descents. I also realized than as soon as the ice had fully melted I had to reduce my exertion level quite a lot or I would quickly feel too hot. 
 
-After 28kms of very hot running, I heard it was just over 32c/90f with a pretty large climb to the end I finally arrived the next crew spot, powerline - wooo, I was roughly half way! 
+After 28kms of very hot running, I heard it was just over 32c/90f with a pretty large climb to the end of this section. I finally arrived the next crew spot, Powerline - wooo, I was roughly half way!
 
 
 ###  Powerline Loop (47.6 to 68.2 miles)
 #### Aid Stations: Powerline to Powerline
 
-As I exited the aid station, I saw Phil again! He had navigated some tricky roads to reach the crew point and was ready to pace me for this 34 km section. I sat down briefly, mumbled about the previous segments while eating and drinking, and looked forward to the next stretch with a bit a bit of company. I also decided to swap my shoes to a fresh pair, which felt great - who knows if the foam had compressed or it was all psychologucal - but I 'm not complaining! After a slightly longer stop, we set out together.
+As I exited the aid station, I saw Phil again! He had navigated some tricky roads to reach the crew point and was ready to pace me for this 34 km section. I sat down briefly, mumbled about the previous segments while eating and drinking, and looked forward to the next stretch with a bit a bit of company. I also decided to swap my shoes to a fresh pair, which felt great - who knows if the foam had compressed or it was all psychological - but I'm not complaining! After a slightly longer stop, we set out together.
 
 The start of the Powerline section was a gentle trail, perfect for making good time. We ran the flats and downhills, power hiking the ups. About 37 minutes in, we spotted the first-place runner, Brendon, powering up a hill. He looked like a man on a mission, and Phil and I were both impressed. We pressed on and reached the next aid station, where we topped up with ice and enjoyed some fantastic pumpkin chocolate cake, definitely the best snack of the day.
 
@@ -122,7 +122,7 @@ The return leg felt quicker, helped by the steady stream of runners heading out.
 
 It was getting dark as we returned to Powerline. Phil and I had a plan: instead of pacing me through the next section, he would meet me for the last 25 km, ensuring I had company for the most challenging part, the finish section.
 
-As I entered the Powerline aid station in for the second time, I couldn’t help but admire those just starting this loop. It had taken me just under 5 hours to complete the loop, and just over 14 hours total race time. If they matched my pace, they’d be over 19 hours in by the time they returned, painful.
+As I entered the Powerline aid station for the second time, I couldn’t help but admire those just starting this loop. It had taken me just under 5 hours to complete the loop, and just over 14 hours total race time. If they matched my pace, they’d be over 19 hours in by the time they returned, painful.
 
 As I prepared to head out, we realized that our head torches had gotten mixed up in the wrong bags. After a momentary panic, we worked out a plan: I left with Phil’s torch, and he would bring mine for the next pacing section. Crisis averted!
 
@@ -136,7 +136,7 @@ From there, I focused on grinding out the miles. The terrain was a mix of climbs
 
  Just as I was about to leave, another shirtless runner bent over and let out a massive chunder. I had to look away to avoid triggering my own stomach!
 
-Not long after, the same runner and his pacer caught up to me. We exchanged pleasantries, and he asked, “Having fun yet?” I replied with a hearty “f@#k no, I just want to get this done.” Various runners and I began leap frogging, many had pacers and sometimes I would try hang on for a few miles using them to drive a bit of extra motivation in myself.
+Not long after, the same runner and his pacer caught up to me. We exchanged pleasantries, and he asked, “Having fun yet?” I replied with a hearty “f@#k no, I just want to get this done”. Various runners and I began leap frogging, many had pacers and sometimes I would try hang on for a few miles using them to drive a bit of extra motivation in myself.
 
 At one point, I teamed up with two guys and their mom, who were setting a great pace. We chatted about San Diego and various other races. Midway through, a runner coming the other way (presumably a sweeper) warned us about a copperhead rattlesnake up ahead. From then on, every snake-shaped branch put us on edge. Eventually, I let them go and tackled the remaining miles on my own.
 
@@ -146,17 +146,17 @@ At one point, I teamed up with two guys and their mom, who were setting a great
  
 Arriving back at the aid station, I thought I had completed about 144 km in 18.5 hours. But my GPS must have been off, or I had been running poor tangents—there were still 26 km to go! It’s a bit disheartening to realize you’re not as far along as you thought. Though on a positive note, Phil would be re-joining me until the end.
 
-After more broth and chips, Phil and I left the aid station. I mentioned the snake story, and he didn't seem to register it. Looking back I guess he was thinking about the next section or something else as not 10 minutes later..... Not 10 minutes later, he nearly stepped on a small copperhead, missing it by just a few inches! He also encountered his first tarantula soon after. Quite the introduction to pacing in Arkansas!
+After more broth and chips, Phil and I left the aid station. I mentioned the snake story, and he didn't seem to register it. Looking back I guess he was thinking about the next section or something else as not 10 minutes later..... he nearly stepped on a small copperhead, missing it by just a few inches! He also encountered his first tarantula soon after. Quite the introduction to pacing in Arkansas!
 
-This next section began with a runnable trail, followed by a rocky climb. It was Phil’s first experience with the trickier terrain, and he remarked on how challenging it was. We kept pressing on, sumitting climbs and then descending them trying to run as much as possible, mixing in a fair amount of death marching. I had long since given up on the top 10 goal and was now focusing on sub-24. With plenty of time in the bank, I focused on maintaining a decent pace to secure that coveted colorful buckle.
+This next section began with a runnable trail, followed by a rocky climb. It was Phil’s first experience with the trickier terrain, and he remarked on how challenging it was. We kept pressing on, summiting climbs and then descending them trying to run as much as possible, mixing in a fair amount of death marching. I had long since given up on the top 10 goal and was now focusing on sub-24. With plenty of time in the bank, I focused on maintaining a decent pace to secure that coveted colorful buckle.
 
-As we descended the final long hill, I knew we’d done it. Phil went ahead to film my finish, and with a final burst of energy, I crossed the line. Hooray! 22 hours and X minutes, my third fastest 100-miler!
+As we descended the final long hill, I knew we’d done it. Phil went ahead to film my finish, and with a final burst of energy, I crossed the line. Hooray! 22 hours and 26 minutes, my third fastest 100-miler, and 24th overall!
 
 After a brief chat with the race director and receiving my buckle, we headed back to the hotel for a well-earned rest.
 
-Epilogue
+### Epilogue
 
-Back at the hotel, we grabbed a terrible McDonald's breakfast, I showered briefly before going down for a well earned nap. Waking up I briefly considered going back to the race to watch the final finishers and the award ceremony,but we were tight on time to return the rental car in Little Rock, so I had to skip it.
+Back at the hotel, we grabbed a terrible McDonald's breakfast, I showered briefly before going down for a well earned nap. Waking up I briefly considered going back to the race to watch the final finishers and the award ceremony, but we were tight on time to return the rental car in Little Rock, so I had to skip it.
 
 Reflecting on the race and my goals, I fell short of my aspirations but knew they were lofty. I let go of my time goals early on, except for finishing under 24 hours, due to the heat. Did I give up on the top 10 too soon? Could I have pushed deeper into the pain cave earlier? Should I have taken more risks later on? I’m not sure. But either way, it was fun to push harder, run more hills, and I’m incredibly proud of the result.
 
